docs/application/native/guides/app-management/common-appcontrols.md

```diff
@@ -1809,6 +1809,70 @@ retrieve_media_operation(const char* mime_type)
 }
 ```
 
+### Creates media controller to communicate with application
+
+The media controller operation is supported in mobile and wearable applications.
+To control media application, use the `APP_CONTROL_OPERATION_MEDIA_CONTROLLER` operation with the `server` and `client` launch type.
+To specify various details refer to the extras defined below.
```

Comment:
Global: 
Please rephrase as following:
To specify various event details, see Extra Input and Extra Output.

Please change this in the entire file.

Reply:
The extras include operation, MIME Type, URI, Extra Input, Extra Output and so on. The extras are different for each event. Could you pelease rephrase the sentense "see Extra Input and Extra Output"?
And the people responsible for managing each event and document are different. So I can't change this in the entire file.